test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubemark Master**
```

Comment:
Also like you mentioned, it's possible to combine both (1) and (2) into the same cluster because of how the scripts are setup.  The kubemark pods could land on (1) and use (1) as the kubemark master because the local kube config will be identical to the kubemark config.   I want to focus this document on the intended kubemark infrastructure, but if folks want to use a single kubernetes cluster to land kubemark on, I can document how to do that too.

Perhaps this doc needs the full explanation of both scenarios to clearly define the differences.  I'll try and make it more clear in the next revision. Thanks for the review @wojtek-t.